util/file_util.cc

```diff
@@ -16,7 +16,8 @@ namespace rocksdb {
 
 // Utility function to copy a file up to a specified length
 Status CopyFile(Env* env, const std::string& source,
-                const std::string& destination, uint64_t size) {
+                const std::string& destination, uint64_t size,
+                const bool sync_after_copy) {
```

Comment:
It seems to me that CopyFile() is only used in CheckpointImpl::CreateCheckpoint(). If we always enforce an fsync, Then maybe we don't have to add such a parameter.